src/main/java/de/danielbechler/diff/inclusion/InclusionService.java

```diff
@@ -24,6 +24,7 @@
 
 import java.util.Collection;
 import java.util.LinkedList;
+import java.util.Set;
```

Comment:
Since this file doesn't contain any actual changes anymore, could you please bring it back into its original condition, so it doesn't show up in the diff?